python/mxnet/ndarray/sparse.py

```diff
@@ -195,7 +195,8 @@ def asnumpy(self):
         return self.tostype('default').asnumpy()
 
     def astype(self, dtype, copy=True):
-        """Returns a copy of the array after casting to a specified type.
+        """Return a copy of the array after casting to a specified type.
```

Comment:
what is the change you made, they look the same :(

Reply:
Returns vs Return

Reply:
That was a pydocstyle suggestion